apps/mcp/.env

```
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_STACK_API_URL=
+NEXT_PUBLIC_SERVER_STACK_API_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SERVER_STACK_API_URL key should go before the NEXT_PUBLIC_STACK_API_URL key

(UnorderedKey)